packages/auth/index.ts

```diff
@@ -19,6 +19,6 @@ declare module "next-auth" {
 
 export * from "./security";
 
-export const createHandlers = (isCredentialsRequest: boolean) => createConfiguration(isCredentialsRequest, headers());
+export const createHandlers = (provider: SupportedAuthProvider | "unknown") => createConfiguration(provider, headers());
```

Comment:
Just a question; when is the provider "unknown"? Maybe consider documenting this somewhere? Perhaps developer docs in homarr.dev?